db/Models/models.js

```diff
@@ -0,0 +1,16 @@
+
+const db = require('../connection')
+
+const fetchTopics = ()=>{
+    const qry = "SELECT * FROM topics"
```

Comment:
Be careful with your variable names, `qry` - I understand it's short for query but do make it a little more descriptive or obvious, query for what?